src/outputs/alsa.c

```diff
@@ -1014,6 +1020,14 @@ alsa_init(void)
       offset = 44100 * (offset/abs(offset));
     }
 
+  original_adjust = adjust_period_seconds = cfg_getint(cfg_audio, "adjust_period_seconds");
```

Comment:
Prefix globals with the module name, so alsa_adjust_period_seconds

Reply:
I don't mind doing this. But I'm surprised by this comment. None of the other static global variables in alsa.c have alsa_ prefix. Why this one?